helpers/z_schema.js

```diff
@@ -275,6 +276,33 @@ var liskFormats = {
 
 		return ip.isV4Format(str) || FormatValidators.hostname(str);
 	},
+	/**
+	 * Transaction amount/fee.
+	 *
+	 * @param {Object} value
+	 * @returns {boolean}
+	 */
+	amount(value) {
+		if (value instanceof bignum) {
+			return (
+				value.greaterThanOrEqualTo(0) &&
+				value.lessThanOrEqualTo(global.constants.totalAmount)
+			);
+		}
+		return false;
+	},
+	/**
+	 * Transaction totalAmount, totalFee, reward.
+	 *
+	 * @param {Object} value
+	 * @returns {boolean}
+	 */
+	minAmount(value) {
```

Comment:
Why we cannot use `amount` check for that?

Reply:
@4miners I don't see any JSON schema validation available for amount type http://json-schema.org/latest/json-schema-validation.html, please refer me to any information where it specifies we can use amount, I will be glad to use and update the same.

Reply:
I mean we can use `amount` check instead of `minAmount` - the only difference is `amount` checking also for max amount, which should be fine.